zoltar-core/src/main/java/com/spotify/zoltar/fs/FileSystemExtras.java

```diff
@@ -40,6 +42,21 @@
   private FileSystemExtras() {
   }
 
+
+  /**
+   * Finds the latest date directory in the directory src. It assumes that partitions are
+   * formatted using ISO_LOCAL_DATE (e.g. YYYY-MM-DD).
+   */
+  public static String getLatestDate(String src) throws IOException {
+    return Files.list(Paths.get(src))
+        .filter(p -> Files.isDirectory(p))
+        .map(p -> LocalDate.parse(p.getFileName().toString()))
```

Comment:
What happens if a directory name is not in a date format? Will `parse` throw exception?

Reply:
Yep! Added a test to show that.

Reply:
I'm thinking if it's better to just ignore the bad one? What do other thinks?